app/auth/authenticate_user.rb

```diff
@@ -13,7 +13,7 @@ def call
   attr_accessor :email, :password
 
   def user
-    user = User.find_by_email(email)
+    user = User.find_by(email: email)
```

Comment:
**Code Quality - Nitpick**
Prefer `find_by(email: email)` to `find_by_email` due to deprecation risk.